iina/FFmpegController.h

```diff
@@ -38,6 +38,21 @@
 
 @property(nonatomic) NSInteger thumbnailCount;
 
+/// Initializes and returns an image object with the contents of the specified URL
+///
+/// At this time, the normal [NSImage](https://developer.apple.com/documentation/appkit/nsimage/1519907-init)
+/// initializer will fail to create an image object if the image file was encoded in [JPEG XL](https://jpeg.org/jpegxl/) format.
+/// In older versions of macOS this will also occur if the image file was encoded in [WebP](https://en.wikipedia.org/wiki/WebP/)
+/// format. As these are supported formats for screenshots it is desirable to have an alternative method for creating image objects that
+/// supports these formats so that a screenshot preview can be displayed as is done for other screenshot formats. This method is
+/// intended to be used when the normal AppKit supplied initializer is unable to create an image.
+/// - Attention: This method only supports decoding JPEG XL and WebP encoded images. It is not intended to replace the normal
+///       AppKit supplied initializer.
+/// - Parameter url: The URL identifying the image.
+/// - Returns: An initialized NSImage object or null if the method cannot create an image representation from the contents of the
+///       specified URL.
++ (nullable NSImage *)createNSImageWithContentsOf:(nonnull NSURL *)url;
```

Comment:
```suggestion
+ (nullable NSImage *)createNSImageWithContentsOfURL:(nonnull NSURL *)url;
```

Reply:
Fixed. I was trying to pattern this after [this](https://developer.apple.com/documentation/appkit/nsimage/1519907-init) `NSImage` initializer. But I could not get the Swift name translation to use `contentsOf` for the parameter name. I had to add "With".